tidb-cloud/monitor-new-relic-integration.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Integrate TiDB Cloud with New Relic
+summary: Learn how to monitor your TiDB cluster with the New Relic integration.
+---
+
+# Integrate TiDB Cloud with New Relic
+
+You can configure TiDB Cloud to send metric data of your TiDB clusters to [New Relic](https://newrelic.com/). After that, you can directly view these metrics in your New Relic dashboards.
+
+## Prerequisites
+
+- To integrate TiDB Cloud with New Relic, you must have a New Relic account and a [New Relic API key](https://one.newrelic.com/admin-portal/api-keys/home?). New Relic grants you an API key when you first create a New Relic account.
+
+    If you do not have a New Relic account, sign up [here](https://newrelic.com/signup).
+
+- To edit third-party integration settings for TiDB Cloud, you must have the **Organization Owner** access to your organization or **Project Member** access to the target project in TiDB Cloud.
+
+## Limitation
+
+You cannot use the New Relic integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+
+## Steps
+
+### Step 1. Integrate with your New Relic API Key
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
+2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
+
+    - If you have multiple projects, switch to the target project, and then click **Admin** > **Integrations**.
+    - If you only have one project, click **Admin** > **Integrations**.
+
+3. Click **Integration to New Relic**.
+4. Enter your API key of New Relic and choose the site of New Relic.
+5. Click **Test Integration**.
+
+    - If the test successes, the **Confirm** button is displayed.
+    - If the test fails, an error message is displayed. Follow the message for troubleshooting and retry the integration.
+
+6. Click **Confirm** to complete the integration.
+
+### Step 2. Add TiDB Cloud Dashboard in New Relic
+
+1. Log in to [New Relic](https://one.newrelic.com/).
+2. Go to the [TiDB Cloud dashboard page](TBD) on **Add Data** page in New Relic.
```

Comment:
Waiting for the link to be available on New Relic after it goes live.

Reply:
the [TiDB Cloud dashboard page](TBD) on **Add Data** page?

Reply:
tabs？